requirements.in

```diff
@@ -1,4 +1,4 @@
-tutor<18.0.0
+tutor==17.0.4
```

Comment:
Please, don't do it like that. Just run `pip-compile` from pip-tools to bump the requirements in `requirements.in`

Reply:
pip-compile won't update requirements.txt to latest version for me, unless I spefically pin it, beacuse may be to my understanding the condition would sitll be true if tuor is on 17.0.3 or 17.0.4

Reply:
@snglth dose work like that for you locally?

Reply:
You have to `rm requirements.txt` for pip-compile to properly bump th requirements. I guess existing `requirements.txt` gets in the way one way or another.